client/src/actions/actionCreators.js

```diff
@@ -23,9 +23,16 @@ export function getTest(cb) {
 	};
 }
 
-/**
- * @description Pings the socket method socket/test
- */
+
+export function getCurrentUser() {
+	return (dispatch, prevState) => { // eslint-disable-line
+		api.getCurrentUser()
+			.then(response => {
+				dispatch({ type: 'GET_CURRENT_USER', current: response });
```

Comment:
Maybe rename `current` to `currentUser`. Also maybe the reducer here is trying to set who the current user is. Maybe rename the reducer to `SET_CURRENT_USER`. This **action** fetches the user, but the reducer stores the user into the store.